extensions/amp-ad/0.1/test/test-amp-ad-api-handler.js

```diff
@@ -35,39 +36,87 @@ describe('amp-ad-api-handler', () => {
     const adElement = document.createElement('amp-ad');
     const adImpl = new BaseElement(adElement);
     apiHandler = new AmpAdApiHandler(adImpl, adImpl.element);
-    return createIframePromise().then(c => {
+    return createIframePromise(null, () => {
```

Comment:
It looks like `null, () => {}` are not needed here - a leftover from our attempts to fix tests